### PR TITLE
Suggest upgrading package from PAK to GITHUB_TOKEN

### LINKED ISSUE #6: github token not working
Thank for this awesome example. 

I've got this working on my project here: https://github.com/neuroscout/neuroscout-cli/pull/136
I set it up to build based on Docker Hub builds using the same meta-data.

One thing I noticed is that for some reason, using `{{ secrets.GITHUB_TOKEN }}` did not work for me. Instead, I had to create a personal token from my account, and add it to the repository secrets. 

If so, it may be worth updating the example.

### PR DESCRIPTION
**Pull request by adelavega**
Closes #6 

I think the exact issue is that the first time I created the package was from pushing locally using a personal access token. When the package is created in that manner, the repo is not automatically given permission to push to the package/registry using  `GITHUB_TOKEN`. 

It's hard to confirm 100% because if you've never pushed to the repo, then the package wouldn't exist, and there wouldn't be any package settings to access...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,8 +45,7 @@ that writes the configuration.
 ### 1. Enable Packages
 
 If you want to use the [GitHub package registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry)
-you'll need to follow the instructions there to enable packages for your organization, specifically "public" and "internal" packages should be allowed to be created.
-You'll also want to add a username associated with your GitHub organization to the repository secret `GHCR_USERNAME`
+you'll need to follow the instructions there to enable packages for your organization, specifically "public" and "internal" packages should be allowed to be created. You'll also want to add a username associated with your GitHub organization to the repository secret `GHCR_USERNAME`. If a package already exists for your repo, you may need to [follow instructions](https://docs.github.com/en/packages/managing-github-packages-using-github-actions-workflows/publishing-and-installing-a-package-with-github-actions#upgrading-a-workflow-that-accesses-ghcrio) to updade a package from using personal access tokens to the safer `GITHUB_TOKEN`.
```
Comment (vsoch) [inline README.md:48]:
```suggestion
you'll need to follow the instructions there to enable packages for your organization, specifically "public" and "internal" packages should be allowed to be created. You'll also want to add a username associated with your GitHub organization to the repository secret `GHCR_USERNAME`. If a package already exists for your repository, you may need to [follow instructions](https://docs.github.com/en/packages/managing-github-packages-using-github-actions-workflows/publishing-and-installing-a-package-with-github-actions#upgrading-a-workflow-that-accesses-ghcrio) to update a package from using personal access tokens to the safer `GITHUB_TOKEN` This means that:

 1. If you want to be able to push to the package registry without extra work, don't push a container to the same package namespace that matches the repository before (e.g., from the command line).
 2. If you have already pushed a container to the repository package namespace, or you want to test and don't mind making the manual changes, you will need to [follow these instructions](https://docs.github.com/en/packages/managing-github-packages-using-github-actions-workflows/publishing-and-installing-a-package-with-github-actions#upgrading-a-workflow-that-accesses-ghcrio) to associate the package with your repository.
```

**Review by vsoch — COMMENTED** [review]
I think that's exactly what happened - I'm thinking back to my experience, and for this repo I just tested logging in from the command line (and didn't push) and it was associated without a hitch. For another project I'm working on, I pushed previously and had to go through these same association steps!

I did a few tweaks and added an if/else thing so the user's options are explicitly clear. Let me know what you think.

File: README.md (hunk 1 above)
Comment (adelavega) [reply README.md:48]: Great. Looks good to me.

**Review by vsoch — APPROVED** [review]
Thank you!

**Pull request merged**